tools/c7n_azure/c7n_azure/container_host/host.py

```diff
@@ -408,8 +394,27 @@ def has_yaml_ext(filename):
         return filename.lower().endswith(('.yml', '.yaml'))
 
 
+@click.command(help="Periodically run a set of policies from an Azure storage container against "
+    "a single subscription. The host will update itself with new policies and event subscriptions "
+    "as they are added.")
+@click.option("--event-queue-id", "-q", envvar=ENV_CONTAINER_EVENT_QUEUE_ID, required=True,
+              help="The resource id of the storage account to create the event queue in")
```

Comment:
nit: it might be out of scope, but might be a good time for a breaking change..
Do we want to be consistent with --policy-storage and ask for queue URI instead of resource ID & queue name?

Reply:
These options do seem a bit strange. `--event-queue-id` and `--event-queue-name` are used to create a new queue rather than providing a queue that already exists. 

Do we want to keep that behavior? If so, I think it would make sense to rename `--event-queue-id` to `--storage-account-id`.

Reply:
I'm not sure this has been in an official release yet, so the extent to which compatibility should be given is pretty minimal afaict, reusing the existing cli url semantics would be preferable to separate params per arg and different usage semantics (create).

Reply:
Yeah there is some back story to how weird those parameters are - and it comes down to us needing a Queue URI and a Resource ID (for the weird event hub subscription API) and it being non-trivial to convert a Queue URI into a Resource ID.  

We'd have to enumerate all their storage accounts and string match on the parsed URI hostname.  I think in this case since we really do need the resource ID it is best to just have the user go track it down.  

I always take a URI when we are just using the data client and any URI will work, but this was a special case.